samples/matter/thermostat/README.rst

```diff
@@ -0,0 +1,349 @@
+#. Press **Button 2** to print the thermostat temperature data to the terminal.
+#. Press **Button 1** for six seconds to initiate the factory reset of the device.
+
+The device reboots after all its settings are erased.
+
+.. _matter_thermostat_sensor_testing:
+
+Testing with external sensor
+----------------------------
+
+After building this sample and the :ref:`Matter weather station <matter_weather_station_app>` application and programming each to the respective development kit and Nordic Thingy:53, complete the following steps to test communication between both devices:
+
+1. |connect_terminal_both|
+#. |connect_terminal_both_ANSI|
+#. If devices were not erased during the programming, press the button responsible for the factory reset on each device.
+#. On each device, press the button that starts the Bluetooth LE advertising.
+#. Commission devices to the Matter network.
+   See `Commissioning the device`_ for more information.
+   During the commissioning process, write down the values for the thermostat node ID, the temperature sensor node ID, and the temperature sensor endpoint ID.
+   These IDs are going to be used in the next steps (*<thermostat_node_ID>*, *<temperature_sensor_node_ID>*, and *<thermostat_sensor_endpoint_ID>*, respectively).
+#. Use the :doc:`CHIP Tool <matter:chip_tool_guide>` ("Writing ACL to the ``accesscontrol`` cluster" section) to add proper ACL for the temperature sensor device.
+   Use the following command, with *<thermostat_node_ID>* and *<temperature_sensor_endpoint_ID>* values from the previous step about commissioning:
+
+   .. code-block:: console
+
+      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, \
+      {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": \
+      [{"cluster": 1026, "endpoint": <thermostat_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
+
+#. Write a binding table to the thermostat to inform the device about the temperature sensor endpoint.
+   Use the following command, with *<temperature_sensor_node_ID>* and *<thermostat_sensor_endpoint_ID>* values from the previous step about commissioning:
+
+   .. code-block:: console
+
+      $ ./chip-tool binding write binding '[{"fabricIndex": 1, "node": <temperature_sensor_node_ID>, "endpoint": <thermostat_sensor_endpoint_ID>, \
+      "cluster": 1026}]' <thermostat_node_ID> 1
+
+   (You can read more about this step in the "Adding a binding table to the ``binding`` cluster" in the CHIP Tool guide.)
+
+   The thermostat is now able to read the real temperature data from the temperature sensor device.
+   The connection is ensured by :ref:`matter_thermostat_sample_binding` to Matter's temperature measurement cluster.
+
+#. Press **Button 2** to print the temperature data from the temperature sensor device to the UART terminal.
```

Comment:
This sentence may suggest that we print the values directly from the temperature sensor device which is not true.
```suggestion
#. Press **Button 2** to print the most recent temperature data from the thermostat to the UART terminal.
```